addons/mail/static/src/messaging/entity/follower_subtype_list/follower_subtype_list.js

```diff
@@ -0,0 +1,39 @@
+odoo.define('mail.messaging.entity.FollowerSubtypeList', function (require) {
+'use strict';
+
+const {
+    fields: {
+        many2one,
+    },
+    registerNewEntity,
+} = require('mail.messaging.entity.core');
+
+function FollowerSubtypeListFactory({ Entity }) {
+
+    class FollowerSubtypeList extends Entity {
+
+        //----------------------------------------------------------------------
+        // Private
+        //----------------------------------------------------------------------
+
+        /**
+         * @override
+         */
+        _update(data) {
+            const { follower } = data;
+            if (follower) {
+                this.link({ follower });
+            }
```

Comment:
no else unlink?

Isn't follower actually supposed required here?

Reply:
For what I understand : this is just used at creation, the component is never updated. The link is then removed when the entity is removed

Reply:
@alexkuhn conceptual stuff here, please suggest change or resolve accordingly.

I don't care what we do, just I'm thinking this is another example of "read only" relation after it is created.

Reply:
I wouldn't put an `else`, just because at the moment update of entity fields is excessively permissive and the more code this method has, the harder it is to use/understand/maintain/migrate entity. Intent is to remove `_update()` and make field definition more expressive, so that update of attribute/relations is mostly automated and we shouldn't manually put if/else code like this.